bazel/dependency_imports.bzl

```diff
@@ -6,14 +6,14 @@ load("@bazel_gazelle//:deps.bzl", "gazelle_dependencies", "go_repository")
 load("@build_bazel_rules_apple//apple:repositories.bzl", "apple_rules_dependencies")
```

Comment:
I couldn't figure out what's the difference between `bazel/dependency_imports.bzl` and `bazel/repositories_extra.bzl`, so let me know if I should move `proxy_wasm_rust_sdk_dependencies()` elsewhere.